Homework_3.py

```diff
@@ -0,0 +1,46 @@
+# import os
+# import sys
+
```

Comment:
Unimportant comment:

Please add any manual steps you did for running this code. Ex.g. adding a jar file for connector